domain/services/model.go

```diff
@@ -97,13 +99,14 @@ type PublicKeyImporter interface {
 type ModelServices struct {
 	modelServiceFactoryBase
 
-	clock             clock.Clock
-	modelUUID         model.UUID
-	providerFactory   providertracker.ProviderFactory
-	objectstore       objectstore.ModelObjectStoreGetter
-	storageRegistry   corestorage.ModelStorageRegistryGetter
-	publicKeyImporter PublicKeyImporter
-	leaseManager      lease.ModelLeaseManagerGetter
+	clock                 clock.Clock
+	modelUUID             model.UUID
+	providerFactory       providertracker.ProviderFactory
+	controllerObjectStore objectstore.ModelObjectStoreGetter
```

Comment:
This isn't a model object store, we should either need to rename it or do something else.

Reply:
Agreed. I want to rename it. Confused me no end at the start. Was going to do that separately.

Reply:
Done & Done